client/my-sites/pages/main.jsx

```diff
@@ -81,6 +81,7 @@ class PagesMain extends Component {
 			status: search ? POST_STATUSES.join( ',' ) : postStatus,
 			type: queryType,
 		};
+		const listKey = [ siteId, status, search ].join( `-` );
```

Comment:
We don't seem to need the separator to be a template literal, it can just be a regular string:

```suggestion
		const listKey = [ siteId, status, search ].join( '-' );
```